exchanges/btse/btse_wrapper.go

```diff
@@ -1077,52 +1030,93 @@ func (b *BTSE) GetHistoricCandlesExtended(ctx context.Context, pair currency.Pai
 	return req.ProcessResponse(timeSeries)
 }
 
-func (b *BTSE) seedOrderSizeLimits(ctx context.Context) error {
-	pairs, err := b.GetMarketSummary(ctx, "", true)
+// GetServerTime returns the current exchange server time.
+func (b *BTSE) GetServerTime(ctx context.Context, _ asset.Item) (time.Time, error) {
+	st, err := b.GetCurrentServerTime(ctx)
 	if err != nil {
-		return err
-	}
-	for x := range pairs {
-		tempValues := OrderSizeLimit{
-			MinOrderSize:     pairs[x].MinOrderSize,
-			MaxOrderSize:     pairs[x].MaxOrderSize,
-			MinSizeIncrement: pairs[x].MinSizeIncrement,
-		}
-		orderSizeLimitMap.Store(pairs[x].Symbol, tempValues)
+		return time.Time{}, err
 	}
+	return st.ISO, nil
+}
 
-	pairs, err = b.GetMarketSummary(ctx, "", false)
-	if err != nil {
-		return err
-	}
-	for x := range pairs {
-		tempValues := OrderSizeLimit{
-			MinOrderSize:     pairs[x].MinOrderSize,
-			MaxOrderSize:     pairs[x].MaxOrderSize,
-			MinSizeIncrement: pairs[x].MinSizeIncrement,
+// ExponentPairs returns a map of symbol names which have a Exponent equivalent
+// e.g. PIT-USD will be returned if M_PIT-USD exsits, and SATS-USD if K_SATS-USD exists
+func (m *MarketSummary) ExponentPairs() map[string]bool {
+	pairs := map[string]bool{}
+	for _, s := range *m {
+		if s.Active && s.HasLiquidity() {
+			if e := s.Exponent(); e != "" {
+				pairs[strings.TrimPrefix(s.Symbol, e+"_")] = true
+			}
 		}
-		orderSizeLimitMap.Store(pairs[x].Symbol, tempValues)
 	}
-	return nil
+	return pairs
+}
+
+// Exponent returns the exponent symbol if a market pair has one; e.g. "B", "M", "K"
+// Returns an empty string if no exponent is found
+func (m *MarketPair) Exponent() string {
+	switch {
```

Comment:
suggestion: 
```go
	splits := strings.Split(m.Symbol, "_")
	if len(splits) < 2 {
		return "", nil
	}
	switch splits[0] {
	case "B", "M", "K":
		return splits[1], nil
	default:
		return "", fmt.Errorf("unhandled exponent %s", splits[0])
	}
```
Handles the splits and any other future case if it ever appears we can introduce it. Returns the symbol without prefix which can be filtered out in the caller if its "".

Reply:
Agreed.

Fixed cf562c0